tests/aws/services/lambda_/test_lambda.py

```diff
@@ -523,65 +523,48 @@ def test_ignore_architecture(self, create_lambda_function, monkeypatch, aws_clie
         assert lambda_arch == native_arch
 
     @pytest.mark.skip  # TODO remove once is_arch_compatible checks work properly
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            "$..LoggingConfig",
+            "$..CreateFunctionResponse.LoggingConfig",
+            "$..RuntimeVersionConfig.RuntimeVersionArn",
+        ]
+    )
     @markers.aws.validated
-    def test_mixed_architecture(self, create_lambda_function, aws_client):
-        """Test emulation and interaction of lambda functions with different architectures.
-        Limitation: only works on ARM hosts that support x86 emulation.
+    def test_mixed_architecture(self, create_lambda_function, aws_client, snapshot):
+        """Test emulation of a lambda function changing architectures.
+        Limitation: only works on hosts that support both ARM and AMD64 architectures.
```

Comment:
Interesting limitation. Think we should introduce a marker for this

Reply:
Agree that adding marker is better than the plain skip. Maybe adding an arm marker:

```
only_on_amd64 = pytest.mark.only_on_amd64
only_on_arm64 = pytest.mark.only_on_arm64
```

This special test would then have both markers.

Reply:
It's just visually a bit tricky to distinguish:

`amd64`
`arm64`

Reply:
True but fortunately markers are mostly for automation and filtering purposes, so I'd say that's ok :+1: